ern-container-gen-android/src/AndroidGenerator.ts

```diff
@@ -314,6 +339,11 @@ You should replace "${annotationProcessorPrefix}:${dependency}" with "annotation
     androidDependencies.regular.push(
       `com.android.support:appcompat-v7:${versions.supportLibraryVersion}`,
     );
+    if (isKotlinEnabled) {
+      androidDependencies.regular.push(
+        `org.jetbrains.kotlin:kotlin-stdlib:${versions.kotlinVersion}`,
```

Comment:
It's likely the plugins may end up using Kotlin extensions that are included with Android Jetpack
We can consider including `androidx.core:core-ktx`

Reply:
Thanks @krunalsshah for the suggestion
I took a look at some random popular native modules written in Kotlin, and I don't see any of them using this dependency :
- [react-native-webview build.gradle](https://github.com/react-native-webview/react-native-webview/blob/89fd2db39b6b86e83b0c3c0ff5a31b83b9e564f3/android/build.gradle#L138)
- [react-native-pager-view build.gradle](https://github.com/callstack/react-native-pager-view/blob/fa14630fda5e202df559fd10441a6b0deca59fd1/android/build.gradle#L129)
- [react-native-tor build.gradle](https://github.com/Sifir-io/react-native-tor/blob/9f9f6a23d2a6150e5359656b737629aa1661fa24/android/build.gradle#L129)
- [react-native-file-access](https://github.com/alpha0010/react-native-file-access/blob/784645e5cf9cfb6e0ba3dfe24b7ee45912b4c2fe/android/build.gradle#L128)

While all of them are using stdlib _(duh ... no choice ;P)_
So it doesn't look that common for native modules to use these extensions _(exclusion rather than the norm)_
Given this, it's better not to add this dependency to the container to avoid extra unnecessary overhead. 
If a native module is using these extensions, the dependency on `android.core:core-ktx` should be added to the `dependencies` object of the corresponding plugin configuration in manifest, as any other additional dependencies that the plugin requires.